src/DynamoCoreWpf/ViewModels/Search/NodeSearchElementViewModel.cs

```diff
@@ -123,13 +104,50 @@ public string Description
             get { return Model.Description; }
         }
 
+        public string Class
+        {
+            get
+            {
+                return Model.Categories.Count > 1 ? Model.Categories.Last() : String.Empty;
```

Comment:
No, Categories can't be null, because they are full name of node. E.g. Geometry.Point.ByCoordinates. In this case Categories are "Geometry", "Point", "ByCoordinates". Node always has at least one Category.